src/docs/working-with-actors/creating-actors.md

```diff
@@ -2,39 +2,133 @@
 layout: docs.hbs
 title: Creating your first Actor
 ---
-# Creating your first Actor
-
->**Note**<br/>
+# Creating Actors
+> **Note**<br/>
 Since Akka.NET enforces parental supervision every actor is supervised and (potentially) the supervisor of its children, it is advisable that you familiarize yourself with [Actor Systems](Actor Systems) and [Supervision and Monitoring](Supervision) and it may also help to read [Actor References, Paths and Addresses](Addressing).
 
 ## Defining an Actor class
-Actors in C# are implemented by extending the `ReceiveActor` class and configuring what messages to receive using the `Receive<TMessage>` method.
+Actors are implemented by extending the `UntypedActor` class and implementing the `OnReceive` method. This method takes the message as a parameter.
```

Comment:
Most docs that mention UntypedActor are ported from the very early days.
I agree with @Danthar that ReceiveaActor should be the default for users.
_but_ once C#7 is out, UntypedActor will come back in fashion as we at least get a naive switch type matching there.
But for now, ReceiveActor is the way to go for users.